src/ontology/mondo-edit.obo

```diff
@@ -531789,21 +531769,35 @@ is_a: MONDO:0016120 ! myotonic syndrome
 [Term]
 id: MONDO:0056795
 name: X-linked spermatogenic failure 1
+subset: gard_rare {source="MONDO:GARD"}
 subset: gard_rare {source="GARD:8406", source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: obsoletion_candidate
+subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "DEL CASTILLO syndrome" EXACT [DOID:0050457]
 synonym: "Del Castillo syndrome" RELATED [OMIM:305700]
```

Comment:
The same synonym (though different spelling) is both "exact" and "related". Please update to the correct synonym type (I don't know which one is correct, you'll have to research)

Reply:
@sabrinatoro I am getting myself a bit confused here and could kindly use your help: 

DO has the following: 
<img width="318" alt="Screenshot 2024-10-29 at 11 31 27 AM" src="https://github.com/user-attachments/assets/7d8b3594-a6fc-4636-9245-e0d182fb77e4">

And lists Del Castillo Syndrome and Germinal Cell Aplasia as synonyms of the parent term, Sertoli cell-only syndrome. 

[OMIM:305700](https://omim.org/entry/305700) is Spermatogenic failure, x-linked, 1; SPGFX1 

<img width="586" alt="Screenshot 2024-10-29 at 11 36 06 AM" src="https://github.com/user-attachments/assets/2dc4d24b-560b-433d-ba06-64b6cdac0b4e">

Do you have advice on how to reconcile this?

Reply:
updated to exact synonyms. updated db-xref for DOID:0050457 'Sertoli cell-only syndrome' to MONDO:mondoIsNarrowerThanSource